src/styles/blocks/section-title.scss

```diff
@@ -0,0 +1,50 @@
+.section-title {
+  font-size: 32px;
+  margin: 0;
+  padding: 0;
+  text-align: center;
+  line-height: 100%;
+  font-weight: 700;
+  letter-spacing: -1.5px;
+  color: $color-main-dark;
+  margin-bottom: 48px;
+
+  &--shadow {
+    position: relative;
```

Comment:
there's no shadow

Reply:
![image](https://github.com/mate-academy/layout_landing-page/assets/130221952/f6ff8a71-449f-40f7-98d9-de8ec67686f0)
There's a shadow on mobile version. On tablet and desktop there's none.